src/electionguard/convert.hpp

```diff
@@ -290,6 +290,62 @@ namespace electionguard
     std::chrono::system_clock::time_point timePointFromIsoString(const string &time,
                                                                  const string &format);
 
+    template <typename T>
+    std::vector<std::reference_wrapper<T>>
+    referenceWrap(const std::vector<std::unique_ptr<T>> &input)
```

Comment:
there are a bunch of places in the code where this could be used (e.g. ballot selections and contests) but i stopped short of replacing those things for now to keep the scope narrow